test/show.jl

```diff
@@ -0,0 +1,73 @@
+using HypothesisTests, Base.Test
+
+# based on power_divergence.jl tests
+d = [[762,484] [327,239] [468,477]]
+m = PowerDivergenceTest(d)
+
+@test sprint(show, m) == """Pearson's Chi-square Test
+-------------------------
+Population details:
+    parameter of interest:   Multinomial Probabilities
+    value under h_0:         [0.255231, 0.19671, 0.11594, 0.0893561, 0.193574, 0.14919]
+    point estimate:          [0.276387, 0.175553, 0.118607, 0.0866884, 0.16975, 0.173014]
+    95% confidence interval: Tuple{Float64,Float64}[(0.2579, 0.2955), (0.1571, 0.1947), (0.1001, 0.1378), (0.0682, 0.1058), (0.1513, 0.1889), (0.1545, 0.1922)]
+
+Test summary:
+    outcome with 95% confidence: reject h_0
+    one-sided p-value:           <1e-6
+
+Details:
+    Sample size:        2757
+    statistic:          30.070149095754687
```

Comment:
This one would also benefit from rounding.

Reply:
Yes. See point 2 in my opening comment. The problem is that the output in the "Details" section varies from test to test. So I think I need to adjust this at the individual test level. I think that's another PR.